app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailDestination.kt

```diff
@@ -131,16 +138,14 @@ private fun ContractDetailScreen(
               key = 1,
               contentType = "InsuranceCard",
             ) {
-              val contractCardData = state.contractDetails.contractCardData
+              val contract = state.insuranceContract
               InsuranceCard(
-                chips = contractCardData.chips,
-                topText = contractCardData.title,
-                bottomText = contractCardData.subtitle,
+                chips = contract.createChips(context),
```

Comment:
Hmm I see the convenience that this brings. It is a bit confusing to have to bring the context all the way to the data layer for this.
In other places, we typically create some sort of sealed interface for the scenarios, and do the mapping to the string itself in the ui code, like we do here for example
https://github.com/HedvigInsurance/android/blob/03175abc23b2fa65a96c57f37229676490b55489/app/app/src/main/kotlin/com/hedvig/app/feature/chat/ui/ChatActivity.kt#L172-L173

But here it may be that we need access to this in more than one place.
Is there some other place, other than in the data module for this to live perhaps? And maybe it can also be a `@Composable` function, so that we can use `getString()` in there? Not sure though

Reply:
Thats great